### PR TITLE
`hide-zero-packages` on organization profiles 

### PR DESCRIPTION
**Pull request by yakov116**
Follows #3103 
1. LINKED ISSUES:
   None

2. TEST URLS:
   https://github.com/microsoft (Should **not** get removed)
	https://github.com/babel (Should get removed)

3. SCREENSHOT:
   None

### CODE REVIEW

**Comment by yakov116** [comment]
That second commit needs to be reverted. Will do it soon.

File: source/features/remove-projects-tab.tsx [hunk 1]
```diff
@@ -31,8 +31,8 @@ async function addNewProjectLink(): Promise<void |false> {
 	);
 }
 
-async function getTabCount(tab: Element): Promise<number> {
-	const counter = select('.Counter', tab);
+export default async function getTabCount(tab: Element, selectors: string): Promise<number> {
+	const counter = select(selectors, tab);
```
Comment (fregante) [inline source/features/remove-projects-tab.tsx:35]:
```suggestion
	const counter = select('.Counter, .num', tab);
```

File: source/features/hide-zero-packages.tsx [hunk 2]
```diff
@@ -2,16 +2,20 @@ import elementReady from 'element-ready';
...
 
-async function init(): Promise<void> {
+async function init(): Promise<void | false> {
 	const packagesCounter = await elementReady([
-		'.numbers-summary [href$="/packages"] .num', // `isRepoRoot`
-		'.UnderlineNav-item[href$="?tab=packages"] .Counter' // `isUserProfile`
-	].join(','));
+		'.numbers-summary [href$="/packages"]', // `isRepoRoot`
+		'.UnderlineNav-item[href$="?tab=packages"]', // `isUserProfile`
+		'.orgnav > .pagehead-tabs-item[href$="/packages"]:not(.selected)' // `isOrganizationProfile`
```
Comment (fregante) [inline source/features/hide-zero-packages.tsx:11]: Shouldn't all of them have `:not(.selected)`?

Comment (yakov116) [reply source/features/hide-zero-packages.tsx:11]: Correct the `isUserProfile` since `isRepoRoot` is not a tab

**Comment by yakov116** [comment]
@fregante should we exclude orgRepoWithAccess?

**Comment by fregante** [comment]
What's orgRepoWithAccess?

File: source/features/hide-zero-packages.tsx [hunk 3]
```diff
@@ -2,16 +2,20 @@ import elementReady from 'element-ready';
...
+	].join());
 
-	if (packagesCounter?.textContent?.trim() === '0') {
-		packagesCounter.closest('li, .UnderlineNav-item')!.remove();
+	if (!packagesCounter || await getTabCount(packagesCounter) > 0) {
+		return false;
 	}
+
+	(packagesCounter.closest('li, .UnderlineNav-item') ?? packagesCounter).remove();
```
Comment (fregante) [inline source/features/hide-zero-packages.tsx:18]: Why did you add `?? packagesCounter`? Doesn't seem to ever be needed

Comment (yakov116) [reply source/features/hide-zero-packages.tsx]: Organisations the closest will be null

**Comment by yakov116** [comment]
> What's orgRepoWithAccess?

We should not hide the packages

File: source/features/hide-zero-packages.tsx (hunk 3 above)
Comment (yakov116) [reply source/features/hide-zero-packages.tsx]: Your right, I tested it now, its working just fine
